packages/frontend-app-api/src/wiring/createApp.tsx

```diff
@@ -383,6 +386,38 @@ function createApiHolder(
     factory: () => new LocalStorageFeatureFlags(),
   });
 
+  factoryRegistry.register('static', {
+    api: identityApiRef,
+    deps: {},
+    factory: () => {
+      const appIdentityProxy = new AppIdentityProxy();
+      // TODO: Remove this when sign-in page is migrated
+      appIdentityProxy.setTarget(
```

Comment:
>[!Note]
The search plugin uses Identity API, so created it to provide a default implementation.